nucleus/deploy/client.py

```diff
@@ -426,6 +419,49 @@ def create_model_endpoint(
                 "Endpoint should be one of the types 'sync' or 'async'"
             )
 
+    def edit_model_endpoint(
+        self,
+        endpoint_name: str,
+        model_bundle: Optional[ModelBundle] = None,
+        cpus: Optional[float] = None,
+        memory: Optional[str] = None,
+        gpus: Optional[int] = None,
+        min_workers: Optional[int] = None,
```

Comment:
Maybe this is clearer in the server side PR (I've so far mainly been looking at the abstractions and less so the semantics 😅 ) but if someone omits a field, do you read the existing value and ensure that it's preserved?

Reply:
Yup, that's the intended behavior